### PR TITLE
fix(atenlib): Cast SymInt to INT64 to fix Windows CI

### PR DESCRIPTION
**Pull request by justinchuby**

### CODE REVIEW

File: onnxscript/function_libs/torch_aten/ops/core.py [hunk 1]
```diff
@@ -2178,7 +2179,7 @@ def aten_index_reduce(
 
 # FIXME(#277): Script when attributes can come before inputs
 @torch_op("aten::index_select", trace_only=True)
-def aten_index_select(self: TTensor, dim: int, index: TInt) -> TTensor:
+def aten_index_select(self: TTensor, dim: int, index: IntType) -> TTensor:
```
Comment (fatcat-z) [inline onnxscript/function_libs/torch_aten/ops/core.py:2182]: Why can't we keep TInt here? Looks like they are the same to me.

If we are using Txxx in most of places, I'd like to use the same style.

Comment (justinchuby) [reply onnxscript/function_libs/torch_aten/ops/core.py:2182]: Since mypy requires a TypeVar to appear at least twice in the signature of a generic function, we are forced to use a plain type here.

**Review by fatcat-z — APPROVED** [review]
Approved, thanks!

File: onnxscript/function_libs/torch_aten/ops/core.py (hunk 1 above)
Comment (justinchuby) [reply onnxscript/function_libs/torch_aten/ops/core.py:2238]: Actually, we can probably get away with TInt here. Let me try again and replace as much with TInt.

Comment (justinchuby) [reply onnxscript/function_libs/torch_aten/ops/core.py]: Done

**Review by abock — APPROVED** [review]
